src/kbmod_wf/task_impls/uri_to_ic.py

```diff
@@ -19,21 +21,29 @@ def uri_to_ic(target_uris_file_path=None, uris_base_dir=None, ic_output_file_pat
 
 #! I believe that we can remove the `uris_base_dir` parameter from the function
 #! signature. It doesn't seem to be used in practice.
-def no_uri_to_ic(target_uris_file_path=None, uris_base_dir=None, ic_output_file_path=None, logger=None):
+def uri_to_ic(
+    uris_filepath: str = None,
+    uris_base_dir: str = None,
```

Comment:
@wilsonbb I have an open question about `uris_base_dir`. I don't think that we really need this as an input parameter, but before I yanked it out, I wanted to check to see if I was missing something here.

Reply:
If I recall correctly, this was added because Colin was running the scripts both on klone and on another cluster where he had copied that target uris over (I believe on Monsoon). So he could then run the same script on the same target uris files but then specify a different base directory (the folder where he had copied the data to)

I don't think it's essential to keep in, but maybe not remove it until we settle whether we will be changing the `.lst` files after Dino's work

Reply:
@ColinOrionChandler could you confirm this? Was this parameter in place so that you could run on multiple clusters?